docs/2.0/reference/accountfactory/configurations-as-code.md

```
@@ -318,6 +318,16 @@ account_factory {
   </HclListItemDescription>
 </HclListItem>
 
+### pr_create_token_name
+
+<HclListItem name="pr_create_token_name" requirement="optional" type="string">
+  <HclListItemDescription>
+
+    (GitHub only) The name of your PR create token if different from the default of `PR_CREATE_TOKEN`.
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)